chapter02/jxp/ch2ex1.pl

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use File::Spec;
+use Cwd qw ( abs_path );
+
+my $curdir=File::Spec->curdir();
+foreach my $file ( my @files=<$curdir/*> ){
```

Comment:
Using ``<*>`` is also an older way of reading files. They added the ``glob`` keyword along the way, which may be preferable in newer code.